float.css

```diff
@@ -1,6 +1,7 @@
 *{
```

Comment:
universal selector is a feature that is created. It is not bad when you are applying to it's feature. Take a look at the post from google engineer explain the benefit of using universal selector : http://www.paulirish.com/2012/box-sizing-border-box-ftw/ . Now why we shouldn't use it in general. Mostly, there might be a duplicated styling or if you don't intend to have default styling element as you put in the universal selector. It will be harder for you to find out why the issue is happening. The performance impact is depend on the browser. (mostly IE 6,7). I think you can try to use this technique but I would recommend to use it carefully. Know when you need to use it. The feature is created for reason.